src/daemon/plugin.c

```diff
@@ -1078,14 +1078,66 @@ static int plugin_compare_read_func(const void *arg0, const void *arg1) {
     return 0;
 } /* int plugin_compare_read_func */
 
+/*
+ * Set the time of the first read according to the settings AlignRead and
+ * Interval in collectd.conf.
+ *
+ * The AlignRead value is in seconds. The value modulo 60 determines the second
+ * of a minute. The integer part of the value divided by 60 determines the
+ * minute of an hour. Optionally, decimal places can be used to set the
+ * fraction of a second (up to milliseconds precision).
+ */
+static void plugin_set_first_read_time(read_func_t *rf) {
+
+  double start = rf->rf_ctx.align_read; /* (default is -1.0) */
```

Comment:
Completely unclear naming: why `align_read` becomes `start` ?

Reply:
I renamed the variable.